### DISCUSSION TITLE
Feature Request: Firestore Connector

### DISCUSSION THREAD

**Discussion by jalseth** [lowdefy/lowdefy#503] [Ideas and feature requests]
Hi! Thanks for the interesting project. It would be nice to have a Firestore connector, allowing Lowdefy to interact with that data store, including its listeners.

**Comment by Gervwyk** [comment]
Hi @jalseth 
Thanks! Firestore will be really cool! Would you mind adding it to the [new connections voting board](https://github.com/lowdefy/lowdefy/discussions/309). 

I see Firebase is already up there, I'm not too familiar with the product. Does Firestore and Firebase have the same SDK? 

We'll look into it in any case.

**Reply by gustavopch** [reply]
@Gervwyk Yes, the SDK is the same. Firebase has many services besides Firestore (like Auth, Storage, Realtime Database, Remote Config. Cloud Messaging, etc. which are all consumed through the same SDK) and they're all best used together. Also, they're always releasing new features, so I suggest the connection is as thin and as flexible as possible to prevent lagging behind their releases and missing new features.

**Reply by Gervwyk** [reply]
Hi @gustavopch indeed we always try to make the integration thin and non-opinionated, yet as dev friendly as possible which sometimes results in tradeoffs. I'll investigate firebase in more detail as well as supabase. Would love to support both of them and all features where possible!
